tests/migration_data/LSSTComCam/DRP-v2.json

```diff
@@ -285,8 +290,6 @@
     "recalibrated_star_unstandardized": "recalibrated_star_unstandardized",
     "recalibrated_star_detector": "sourceTable",
     "recalibrated_star": "sourceTable_visit",
-    "isolated_star_predicted_positions": "isolated_star_predicted_positions",
-    "isolated_star_stellar_motions": "isolated_star_stellar_motions",
```

Comment:
I remember seeing a Slack message about this but not the resolution.  Even if you're sure it was a mistake that you're correct to remove on this branch, please put it on a separate commit.

Seems like something @cmsaunders might know about.

Reply:
Oh, looking at this more closely for some reason this was just a case of the `DRP-v2.new.json` suggestions moving these two lines to 307 and 308 below. I'm not sure why it did that but I can revert them.